public/css/styles.css

```diff
@@ -67,41 +75,136 @@ h1 {
   position: relative;
   display: flex;
   justify-content: center;
-  max-width: 400px;
+  width: 100%;
 }
 
 .btn {
-  margin: 1rem;
+  /* margin-top: 1rem; */
   position: relative;
-  min-width: calc(200px - 1rem);
+  width: 100%;
+}
+
+button.btn.btn-info {
+  margin-right: 5px;
+}
+
+a.btn.btn-light {
+  margin-left: 10px;
+}
+
+.form-control {
+  width: 100% !important;
+  margin: 0 !important;
+}
+
+.navbar-collapse {
+  margin-top: 1rem;
 }
 
-/** NAVBAR **/
```

Comment:
In general i think it's nice to keep comments for sections, unless we're organizing alphabetically or in some other way.